ruby/basic_ruby/hashes.md

```diff
@@ -169,7 +169,7 @@ japanese_cars[:honda]   #=> "Accord"
 ### Additional Resources
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
-* Read the article [Ruby Explained: Hashes](http://www.eriktrautman.com/posts/ruby-explained-hashes) by Erik Trautman. One of the delete examples is incorrect: deleting a key by setting its value to nil does not delete the key-value pair.
+* Read the article [Ruby Explained: Hashes](http://www.eriktrautman.com/posts/ruby-explained-hashes) by Erik Trautman. Note that one of the delete examples is incorrect: setting a value to `nil` does not delete the key-value pair. Instead, it sets the key's value to `nil`.
```

Comment:
```suggestion
* Read the article [Ruby Explained: Hashes](http://www.eriktrautman.com/posts/ruby-explained-hashes) by Erik Trautman. Note that one of the delete examples is incorrect: setting a value to `nil` does not delete the key-value pair.
```
I personally don't think the last part is necessary to expand upon. What you changed it to is very matter of fact and explanation enough.